README.md

```diff
@@ -44,6 +44,33 @@ The Oracle WebLogic Server Deploy Tooling is designed to support a wide range of
 - The attribute data types and how to get/set their values (which isn't as easy as it might sound)
 - The differences between WLST online and WLST offline for working with folders and attributes
 
+### Supported WebLogic Server Releases
+The chart below specifies the supported WebLogic Server release, along with the JDK version(s) that must be used to run the WDT tool. You must set the JAVA_HOME environment variable to specify a version of a JDK different than the system default version. 
+
+ To create a domain with the proper JDK (particularly if the JAVA_HOME is different than the one which will be used by the target domain), set the Domain JavaHome attribute in the domain model.
+ 
+ Note that the WDT Encryption tool used to encrypt / decrypt clear passwords in the model and variable file requires WDT to run with a minimum JDK version of 1.8. 
+
+  | WebLogic Server Version | Tool JDK Versions |
+  |-------------------------|-------------------|
+  | 10.3.6                  | 1.7               |
+  | 12.1.1                  | 1.7, 1.8          |
+  | 12.1.2 *, **            | 1.7, 1.8          |
+  | 12.1.3                  | 1.7, 1.8          |
+  | 12.2.1 ***              | 1.8               |
+  | 12.2.1.1 ****           | 1.8               |
+  | 12.2.1.2                | 1.8               |
+  | 12.2.1.3                | 1.8               |
+  | 12.2.1.4 *****          | 1.8               |
+  | 14.1.1                  | 1.8, 1.11         |    
+  
+  \* First release dynamic clusters are supported  
+  \*\* First release coherence clusters are supported  
+  \*\*\* First Release WLS roles are supported  
+  \*\*\*\* First release Multi-Tenancy is supported  
+  \*\*\*\*\* Last release Multi-Tenancy is supported
```

Comment:
Multi-Tenancy -> multitenancy
I suggest that you use footnote numbers (that are links, if possible) to specify the version attributes you describe below the table, instead of asterisks. It's hard to distinguish between the number of asterisks.